plugins/SSH.koplugin/main.lua

```diff
@@ -0,0 +1,154 @@
+local Device =  require("device")
+local InfoMessage = require("ui/widget/infomessage")  -- luacheck:ignore
+local InputDialog = require("ui/widget/inputdialog")
+local UIManager = require("ui/uimanager")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local logger = require("logger")
+local util = require("util")
+local _ = require("gettext")
+
+--This plugin use a patched dropbear that add two things :
+--  the -n option to allow login without password
+--  read the keyfile from settings/SSH/authorized_keys
+
+if not util.pathExists("dropbearmulti") then return { disabled = true, } end
```

Comment:
The comma is a bit weird here to. The normal comma scenario is like this:

```lua
{
  disabled = true, -- no need to do anything to this line! never a syntax error
  -- also, smaller diff
}
```

I personally incline to either space it out properly with comma or a quick one-liner without.

Reply:
I changed it to : 

if not util.pathExists("dropbearmulti") then
    return { disabled = true, }
end

Is this what you mean ?